bionic/deriver.py

```diff
@@ -329,12 +330,11 @@ def _compute_result_group_for_dnode(self, dnode):
             self._get_or_create_task_state_for_key(task.keys[0]) for task in dinfo.tasks
         ]
 
-        task_runner = TaskCompletionRunner(self._bootstrap, self._flow_instance_uuid)
-
-        log_level = logging.INFO if self._bootstrap is not None else logging.DEBUG
-        task_key_logger = TaskKeyLogger(log_level)
+        task_key_logger = TaskKeyLogger(self._bootstrap)
 
+        task_runner = TaskCompletionRunner(self._bootstrap, self._flow_instance_uuid)
```

Comment:
I initially thought of keeping `task_key_logger` inside `TaskCompletionRunner` but decided against it precisely because we were using it at the end of the deriver method. I just didn't think of using `task_runner.task_key_logger` (which sounds so obvious now). I'll make this change.